### PR TITLE
Update version number and align with upstream standards

### PR DESCRIPTION
**Pull request by NigelSomerfieldPNZ**
Update version numbering to v3.0.0-rc2, and align with OIDC upstream standards (include  additional claims, set `"additionalProperties": true`)

### CODE REVIEW

**Review by fabiancook — APPROVED** [review]

File: working/v3.0.0-rc2/hybrid-flow/authorization-request-schema.json [hunk 1]
```diff
@@ -131,6 +131,35 @@
...
+      "examples": ["none", "login consent", "consent", "select_account"]
+    },
+    "ui_locales": {
+      "$id": "#/properties/ui_locales",
+      "type": "string",
+      "title": "Specify preferred languages to display authentication and consent to customer",
+      "examples": ["en-nz", "mi", "en-nz mi"]
+    },
+    "id_token_hint": {
+      "$id": "/properties/id_token_hint",
```
Comment (fabiancook) [inline working/v3.0.0-rc2/hybrid-flow/authorization-request-schema.json:153]: Spotted this after clicking approve sorry, this I am pretty sure should be `#/properties/id_token_hint` inline with the other `$id` values.

**Review by fabiancook — CHANGES_REQUESTED** [review]
```json
"id_token_hint": {
      "$id": "/properties/id_token_hint"
```
      
Should be 

```json
"id_token_hint": {
      "$id": "#/properties/id_token_hint"
```

File: working/v3.0.0-rc2/hybrid-flow/authorization-request-schema.json (hunk 1 above)
Comment (NigelSomerfieldPNZ) [reply working/v3.0.0-rc2/hybrid-flow/authorization-request-schema.json]: Good spotting, I've corrected that now and done a push to update

File: working/v3.0.0-rc2/ciba-flow/CIBA-authorization-request-object-schema.json [hunk 2]
```diff
@@ -104,6 +104,21 @@
...
+    },
+    "user_code": {
+      "$id": "/properties/user_code",
+      "type": "string",
+      "description": "A secret code, such as a password or pin, that is known only to the user but verifiable by the authorisation server"
+    },
+    "request_context": {
+      "$id": "/properties/request_context",
+      "type": "object",
+      "description": "A JSON object containing information to inform fraud and threat decisions.  See CIBA FAPI."
```
Comment (fabiancook) [inline working/v3.0.0-rc2/ciba-flow/CIBA-authorization-request-object-schema.json:121]: These three here (`binding_message`, `user_code `, `request_context `) too should be `"$id": "#/...`

**Review by fabiancook — APPROVED** [review]

File: working/v3.0.0-rc2/ciba-flow/CIBA-authorization-request-object-schema.json (hunk 2 above)
Comment (NigelSomerfieldPNZ) [reply working/v3.0.0-rc2/ciba-flow/CIBA-authorization-request-object-schema.json:121]: Yup, sorry - got a bit enthusiastic and pushed before I'd updated all missing `#`

**Comment by WestpacApiSquad-ContactDavidGliddon** [comment]
Look fine to me

**Review by rvbabilonia — APPROVED** [review]

**Review by DavidGliddon — APPROVED** [review]

**Pull request merged**